themes/cakephp/static/cake.css

```diff
@@ -337,6 +337,18 @@ a.improve {
 #sidebar-navigation > ul > li:lang(fr):nth-of-type(1):before {
     content: "Préface";
 }
+#sidebar-navigation > ul > li:lang(es):nth-of-type(1):before {
```

Comment:
I don't see much values in including these if they are not translated as the english verison will be used anyway, and it add extra maintenance...